### PR TITLE
people doing dissections get paid with rnd funds

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: people doing dissections get paid with sci funds with a 0.3 multiplier
/:cl:

it doesnt generate money, gives incentives to people helping rnd and works in cryptocurrency based societies

### CODE REVIEW

**Comment by MacBlaze1** [comment]
* comments valid criticism* gets blocked 

s-sad!! No but seriously this is a good idea, medical doctors need love

**Comment by ExcessiveUseOfCobblestone** [comment]
Research shouldn't be a science exclusive activity though 😏

**Comment by ExcessiveUseOfCobblestone** [comment]
FYI I haven't merged this because RND shouldn't be considered a science activity. The plan is still (to my knowledge) to move it out of science, it just hasn't been done yet.

File: code/modules/surgery/experimental_dissection.dm [hunk 1]
```diff
@@ -78,6 +78,21 @@
 	target.apply_damage(80, BRUTE, L)
```
Comment (PKPenguin321) [inline code/modules/surgery/experimental_dissection.dm]: I kind of hate that all of this code is ran in the surgery itself, seems too magical to have your ID automagically paid. There should be a Claim Credits button on the surgery console that's created when the surgery is finished, and can be swiped by anybody.

**Review by PKPenguin321 — CHANGES_REQUESTED** [review]

**Comment by PKPenguin321** [comment]
>FYI I haven't merged this because RND shouldn't be considered a science activity.

I had assumed this would be a medbay activity, not an RND one, so it should totally fit within this idea

**Comment by ExcessiveUseOfCobblestone** [comment]
Medbay will typically do the surgery (maybe robros do it too, idk) but the act of progressing via RND is a stationwide activity, not a science activity. It doesn't make sense for it to automatically pull out of science funds when they aren't (SUPPOSE TO BE, at least) managing the flow of research.

It's just no one has changed it yet 😏

**Comment by ExcessiveUseOfCobblestone** [comment]
Staling this until someone says "no we no longer want to pull RND out of science"

**Comment by Tlaltecuhtli** [comment]
the funds are called "research funds" and the research director holds the research funds, he has all interest into pay people to do research, it makes sense they are paid by research funds

**Comment by ExcessiveUseOfCobblestone** [comment]
> the funds are called "research funds" /// research director holds the research funds

I don't care what it's called, it's the science department budget.

> he has all interest into pay people to do research

That's because currently he controls the flow of what gets unlocked for the purpose of techwebs. The proper fix is to move it out of research.

> it makes sense they are paid by research funds

because rnd is currently restricted to science. It will no longer make sense to pull from the science budget when unlocking content via the rnd computer is not bound to what scientists like/don't like as was the intention of techwebs.

Personally I think it would be cool to not have all of the RP go to the cloud, but instead a portion of it goes on the person('s card?), then they can trade it as a secondary currency (with the intention these points can only be accessed by the person holding them). 

That would also encourage use since it feels like you personally get rewarded instead of doing it for the benefit of others (even though for the most part you still are).

**Comment by Tlaltecuhtli** [comment]
you want an item that gets turned into rnd poins?

**Comment by ExcessiveUseOfCobblestone** [comment]
not necessarily a physical representation like holochips, but I'd like a localized version of RP points that the user has control over and therefore can trade with/for to reserve points.

Basically you'd have the public pool (what we have now) that most of the points go to, but you'd also have your local pool that can only be used by you until you donate/trade it.

This is on top of throwing RND out of science so other heads can use the machine.

**Pull request closed without merging**